src/containers/my-cooperations/my-cooperations-details/MyCooperationsDetails.tsx

```diff
@@ -86,6 +88,9 @@ const MyCooperationsDetails = () => {
       </Box>
     ))
 
+  const cooperationCompletion =
+    userRole === UserRoleEnum.Tutor ? <CooperationCompletion /> : null
```

Comment:
It's a good solution, I'm not against it. But I write this line of code like this:
```suggestion
    userRole === UserRoleEnum.Tutor && <CooperationCompletion />
```